.gitignore

```diff
@@ -20,11 +20,14 @@ teleproxy
 kat/kat/client
 /.docker_port_forward
 
-/kat-server-docker-image/kat-server
-/kat-sandbox/grpc_auth/docker-compose.yml
-/kat-sandbox/grpc_web/docker-compose.yaml
-/kat-sandbox/grpc_web/*_pb.js
-/kat-sandbox/http_auth/docker-compose.yml
+# binaries
+/cmd/ambex/ambex
+
+/build/kat/server/kat-server
```

Comment:
Should the KAT backend be in `cmd` as well?

Reply:
The `kat` code is located in `/cmd`, but the binary is left in `/build/kat/server/kat-server`. You can see the build rule:

`
build/kat/server/kat-server: $(wildcard cmd/kat-server/* cmd/kat-server/*/*) pkg/apis/kat/echo.pb.go
	GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -o $@ ./cmd/kat-server
`